### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by NoPodi**

### CODE REVIEW

File: talks/2017_04_05_DependencyManagement/readme.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Presenting a simple demonstration for a web-development stack with **node.js (np
 
 ### Have fun!
 
-## Userful *Visual Studio Code* Extensions
```
Comment (DDCreationStudios) [inline talks/2017_04_05_DependencyManagement/readme.md]: Confusing!

Comment (NoPodi) [inline talks/2017_04_05_DependencyManagement/readme.md]: but it's so clear!

**Comment by robeerob** [comment]
you guys are confusing me :s

**Comment by DDCreationStudios** [comment]
Don't worry @robeerob ! We are just testing and playing :P

**Comment by robeerob** [comment]
That was thought ;)

**Pull request merged**